README.md

```diff
@@ -1,136 +1,134 @@
 
-See my [blog article](https://oren.codes/2017/01/04/multi-targeting-the-world-a-single-project-to-rule-them-all/) for some background on how to get started. Installing
-this package, `MSBuild.Sdk.Extras` adds the missing properties so that you can use any TFM you want.
+The primary goal of this project is to enable multi-targeting without you having to enter in tons of properties within your `csproj`, `vbproj`, `fsproj`, thus keeping it nice and clean.
 
-In short: Create a new .NET Standard class library in VS 2017. Then you can edit the `TargetFramework` to a different TFM (after installing this package), or you can rename `TargetFramework` to `TargetFrameworks` and specify multiple TFM's with a `;` separator.
+See the [blog post](https://oren.codes/2017/01/04/multi-targeting-the-world-a-single-project-to-rule-them-all) for more information.
 
-After building, you can use the `Pack` target to easily create a NuGet package: `msbuild /t:Pack ...`
+### Package Name: `MSBuild.Sdk.Extras`
 
-Few notes:
+[![MSBuild.Sdk.Extras](https://img.shields.io/nuget/v/MSBuild.Sdk.Extras.svg)](https://nuget.org/packages/MSBuild.Sdk.Extras)
+[![MSBuild.Sdk.Extras](https://img.shields.io/myget/msbuildsdkextras/v/MSBuild.Sdk.Extras.svg)](https://myget.org/feed/msbuildsdkextras/package/nuget/MSBuild.Sdk.Extras)
+[![MSBuildSDKExtras](https://img.shields.io/badge/MSBuildSDKExtras-myget-brightgreen.svg)](https://myget.org/gallery/msbuildsdkextras)
 
-- This will only work in VS 2017, Visual Studio for Mac. It's possible it will work in Code, but only as an editor as this requires full `msbuild` to build.
-- To compile, you'll need the desktop build engine -- `msbuild`. Most of the platforms rely on tasks and utilities that are not yet cross platform
-- You must install the tools of the platforms you intend to build. For Xamarin, that means the Xamarin Workload; for UWP install those tools as well.
+### Getting started (VS 15.6+)
 
-NuGet: `MSBuild.Sdk.Extras`
+Visual Studio 2017 Update 6 (aka _v15.6_) includes support for SDK's resolved from NuGet. That makes using the custom Sdks much easier.
 
-[![MSBuild.Sdk.Extras](https://img.shields.io/nuget/v/MSBuild.Sdk.Extras.svg)](https://nuget.org/packages/MSBuild.Sdk.Extras)
+#### Using the SDK
 
-MyGet CI feed: `https://myget.org/F/msbuildsdkextras/api/v3/index.json`
+1. Create a new project
+    - .NET Core console app or .NET Standard class library.
+    - With your exisiting SDK-style project.
+    - With the templates in the repo's [TestProjects](/TestProjects) folder.
 
-[![MSBuild.Sdk.Extras](https://img.shields.io/myget/msbuildsdkextras/v/MSBuild.Sdk.Extras.svg)](https://myget.org/gallery/msbuildsdkextras)
+2. Replace `Microsoft.NET.Sdk` with `MSBuild.Sdk.Extras` to the project's top-level `Sdk` attribute.
```

Comment:
This only works if there's a global.json with the SDK version in it. otherwise you need the /1.3.4 syntax.